rstream/client.py

```diff
@@ -172,9 +180,22 @@ async def start(self) -> None:
         self._conn = Connection(self.host, self.port, self._ssl_context)
         await self._conn.open()
         self.start_task("listener", self._listener())
+        self.start_task("run_delivery_handlers", self._run_delivery_handlers())
         self.add_handler(schema.Heartbeat, self._on_heartbeat)
         self.add_handler(schema.Close, self._on_close)
 
+    async def _run_delivery_handlers(self):
+
+        while True:
+            try:
+                frame_entry = await self._frames.get()
+                maybe_coro = await frame_entry.handler(frame_entry.frame)
+                if maybe_coro is not None:
+                    await maybe_coro
+            except Exception as e:
+                logger.debug("Error reading item from _run_delivery_handlers: " + str(e))
```

Comment:
That seems to be a critical error, probably shouldn't be `debug` level. Also, currently we're just silently stopping `_run_delivery_handlers` task without raising any error, it's unclear what the user should do about it.

This try/except wraps both library code and user-provided callback, which is why it's difficult to handle errors carefully. But I think at least we should log an error message (so the user can get a monitoring alert) and if we choose to suppress such kind of errors, the loop shouldn't be stopped:
```python
while True:
    try:
        ...
    except Exception:
        logger.exception("Error while handlilng %s frame", frame.__class__)
        # no break
```